### DISCUSSION TITLE
Change the default level in plotIPDF

### DISCUSSION THREAD

**Discussion by psaidi** [mtex-toolbox/mtex#1018] [Ask Anything] [answered]
Hi there, 
I am using the following commends to plot the inverse pole figure from Euler angles. 

cs = crystalSymmetry('cubic')
ori = orientation.load(fname,'columnNames', { 'phi1' 'Phi' 'phi2' 'x' 'y'}, 'Columns', [1 2 3 4 5],cs, 'radians')
r1 = vector3d.Z;
plotIPDF(ori,r1,'contourf')

It works well, however, I do not know how to add levels to it. Instead of the default values I want to change the number of levels in the contour and specify them. Would you please help me to do so. 

Thank you, 
Peyman

**Comment by kilir** [comment, marked as answer]
Hi,
you can set the option `'contours',[1 2 3 4]` (or whatever number suits) within plotIPDF which should do what you want. Alternatively, havea look at the function `CLim()`. 
Hope that helps.
Cheers,
rüdiger
